### ISSUE TITLE
sapi5 executable windows installer 

### ISSUE THREAD

**Issue by joshknnd1982** [espeak-ng/espeak-ng#966]
Is there an executable installer for the latest espeak-ng that will install it onto windows10 for both 32bit and 64-bit sapi5? If not can the developers please create one, thanks.

**Comment by sukiletxe** [comment]
Yes, there is, though SAPI won't work properly.

**Comment by joshknnd1982** [comment]
contents of new intonations file pasted below and voice variant is below that… But it is not working the new tunes statements are ignored after being compiled with espeakedit, no confirmation of successful compile is given. 

tune s5
prehead   70 70
headenv   fall 20
head       70 60 70 55 50
headextend 50 55 65 55 40
nucleus0 fall 70 50
nucleus  fall 70 65 60 40
endtune

tune c5
prehead   60 70
headenv   fall 20
head       60 65 70 60 55
headextend 50 60 65 65 40
nucleus0 fall-rise  60 70
nucleus  fall-rise2 70 50 60 65
endtune

tune q5
prehead   65 65
headenv   fall 5
head       50 55 65 70 75
headextend 55 60 65 70 50
nucleus0 fall-rise  60 80
nucleus  fall-rise2 55 65 70 75
endtune

tune e5
prehead   60 70
headenv   fall 10
head       50 55 65 75 50
headextend 45 55 70 70 50
nucleus0 fall  70 50
nucleus  fall  70 60 55 50
endtune

variant file for grossGang sapi5 espeak…

language variant
name testvoice
klatt 5
voicing 100
consonants 90 90 

formant 1 92 100 130
formant 2 103 100 80
formant 3 103 100 70
formant 4 114 100 60

tunes s5 c5 q5 e5


Sent from Mail for Windows 10

**Comment by joshknnd1982** [comment]
I want to play with klatt parameters to make child voices, brighter or more dull voices with more or less trebble and bass, increase and decrease headSize etcetera. Modding tunes statements will let me make Espeak simulate the speaking style of other tts engines such as decTalk and Keynote gold and doubletalk. 


Sent from Mail for Windows 10

**Comment by hozosch** [comment]
On windows 10, you need to run Espeakedit as admin for the compiling to work.

**Comment by joshknnd1982** [comment]
and what do I need to edit in order to modify how the clot voices sound? klatt voice formants

**Comment by hozosch** [comment]
Yes, it's the formants you need to modify for the sound of the voice, not for the intonation though.

**Comment by joshknnd1982** [comment]
what file or files do I need to modify in order to change the formants?

**Comment by hozosch** [comment]
Don't you know how to create ESpeak variants? If you mean adjusting things in the Speech Player engine itself though, I can't help you with that either.

**Comment by joshknnd1982** [comment]
yes I do know how to create a speak variance. I will have to experiment with those first and I may not even need to modify the speech player engine itself.
